e2e/suite_test.go

```diff
@@ -76,12 +76,24 @@ func beforeSuite(ctx context.Context) {
 		ginkgo.By("re-using test environment")
 	}
 
+	// Set timeout for services to become ready
+	var servicesReadyTimeout time.Duration
+	switch cfg.TestEnvConfig.Platform {
+	case types.EnvironmentTypeDocker:
+		servicesReadyTimeout = 5 * time.Second
+	case types.EnvironmentTypeAWS:
+		servicesReadyTimeout = 10 * time.Minute
+	case types.EnvironmentTypeGCP, types.EnvironmentTypeAzure, types.EnvironmentTypeKubernetes:
+	default:
+		servicesReadyTimeout = time.Minute
+	}
+
```

Comment:
Shall this be a helper function the same way as the `getDefaultScanTimeout` and `getDefaultScanTimeout`?

Probably I'd combine these 3 parameter into a single struct generated by a single helper function based on the `types.EnvironmentType`.

```go
type TestSuiteParams struct {
	Scope string
	ServiceReadynessTimeout time.Duration
	ScanTimeout time.Duration
}

func TestSuiteParamsForEnv(t types.EnvironmentType) *TestSuiteParams {
	p := &TestSuiteParams{}
	scope := "assetInfo/%s/any(t: t/key eq 'scanconfig' and t/value eq 'test')"

	switch t {
	case types.EnvironmentTypeAWS, types.EnvironmentTypeGCP, types.EnvironmentTypeAzure:
		...
	case types.EnvironmentTypeDocker, types.EnvironmentTypeKubernetes:
		...
	default:
		...
	}

	return p
}
```

Reply:
I like this idea too, have all the env-specific configs flow through a shared struct